InvenTree/company/models.py

```diff
@@ -250,6 +250,78 @@ def get_api_url():
     role = models.CharField(max_length=100, blank=True)
 
 
+class Address(models.Model):
```

Comment:
Id suggest renaming this to CompanyAddress, what do you think @matmair ? Can this be also used for other models?

Reply:
It's part of the Company app, so figured Address would work fine

Reply:
I think Adress is fitting; no strong feeling about it though